<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Recorridos</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="preOrdenPushButton">
           <property name="text">
            <string>PreOrden</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="postOrdenPushButton">
           <property name="text">
            <string>PostOrden</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="inOrdenPushButton">
           <property name="text">
            <string>InOrden</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Acciones</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QSpinBox" name="insertarSpinBox"/>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="insertarPushButton">
             <property name="text">
              <string>Insertar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QSpinBox" name="eliminarSpinBox"/>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="eliminarPushButton">
             <property name="text">
              <string>Eliminar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="limpiarPushButton">
           <property name="text">
            <string>Limpiar vista</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="vaciarPushButton">
           <property name="text">
            <string>Vaciar Ã¡rbol</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="listView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
